src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -1536,7 +1536,6 @@ private void setupPathSearcher()
                 if (_canDoPathLookupResult == CanDoPathLookupResult.Yes)
                 {
                     _canDoPathLookup = true;
-                    _commandName = _commandName.TrimEnd(Utils.Separators.PathSearchTrimEnd);
```

Comment:
Please revert changes related to `TrimEnd` since it was moved to https://github.com/PowerShell/PowerShell/pull/18166.

Reply:
Done.